src/tests/user.test.ts

```diff
@@ -0,0 +1,26 @@
+import App from '@/app';
+import request from 'supertest';
+import { UserRoute } from '@/routes';
+import { pgDataSource } from '@databases';
+import { AuthService } from '@/services';
+import { BaseUserResponseDTO } from '@/dtos';
+const userRoute = new UserRoute();
+const app = new App([userRoute]);
+const authService = new AuthService();
+
+const testCookie = authService.createTestCookie();
+beforeEach(async () => {
+  await pgDataSource.initialize();
+});
+
+afterEach(async () => {
+  await pgDataSource.destroy();
+});
+
+describe('Testing User Endpoints', () => {
```

Comment:
Maybe we can be more specific here and make the test more descriptive.  Maybe something like "Testing get users endpoint"

Reply:
I think it's fine, since we're building more than just get user endpoints